src/view/AktivitetskortInfoBox.tsx

```diff
@@ -1,31 +1,27 @@
 import React from 'react';
-import { Aktivitet } from '../utils/typer';
+import { Aktivitet, AktivitetTypes, ArenaAktivitet, ArenaAktivitetTypes } from '../utils/typer';
 import { EtikettLiten, Undertekst } from 'nav-frontend-typografi';
 import { formaterDate } from '../utils/date';
 
 import Lenke from 'nav-frontend-lenker';
 
 interface Props {
-    aktivitet: Aktivitet;
+    aktivitet: Aktivitet | ArenaAktivitet;
 }
 
 export function AktivitetskortInfoBox(props: Props) {
-    const datapunkter: Array<Array<InfoElement>> = fjernTommeRaderOgKolonner(mapAktivietTypeToInfobox(props.aktivitet));
+    const datapunkter: Array<InfoElement> = fjernTommeRaderOgKolonner(mapAktivietTypeToInfobox(props.aktivitet));
 
     return (
         <>
             {datapunkter.map(rad => (
-                <div className="aktivitet-kort__infobox__row">
-                    {rad.map(element => (
-                        <div className={'aktivitet-kort__infobox__row__item'}>
-                            <EtikettLiten children={element.label} className="label" />
-                            {element.label === 'Lenke' ? (
-                                <Lenke href={element.data} children={element.data} />
-                            ) : (
-                                <Undertekst children={element.data} className="data" />
-                            )}
-                        </div>
-                    ))}
+                <div className={'aktivitet-kort__infobox__row__item'}>
```

Comment:
få dette ut i en egen komponent?

Umulig å lese

Reply:
Har ikke sett så nøye på det enda, har bare endra fra array of array til array. Kommer mer i neste pr